packages/main/src/index.ts

```diff
@@ -56,21 +92,13 @@ if (isWindows) {
   app.setAppUserModelId(app.name);
 }
 
-/**
- * Check new app version in production mode only
- */
-if (import.meta.env.PROD) {
-  app
```

Comment:
it seems you removed the fact to enable this behavior only for production builds ?

Reply:
still WIP, was removed for testing, latest PR pushed adds this back.

I'll ping once ready for review!